src/NuGet.Core/NuGet.Versioning/NuGetVersion.cs

```diff
@@ -184,5 +184,13 @@ public int Revision
         {
             get { return _version.Revision; }
         }
+
+        /// <summary>
+        /// Returns true is version is a semver2 version
```

Comment:
typo after true, here also it should probably use the official SemVer 2.0.0 name